server/__init__.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+from server.constants import CONSTANTS
+from server.controllers.abc import DatabaseDriver
+from server.controller.mongo import MongoDriver
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+app = Flask(__name__, static_folder='build')
+db: DatabaseDriver = None
+
+if CONSTANTS['ENV'] == 'development':
```

Comment:
I would make this an env on it's own. Generally try to avoid all encompassing vars like `ENV` cause we might want CI at some point that will also need the db but we want other settings to be closer to PROD.

Reply:
That's just the name of the variable internally, externally it grabs that from `FLASK_ENV`

Reply:
Oh wait I'm dumb I see what you mean

Reply:
Yea but what I mean is having one variable control multiple behaviours isn't ideal. At the moment the `ENV` var only controls what db driver is used so it's fine but it's name suggests it would control other behaviour. I think calling this something like `db_driver` and you simply assign it to the driver name you want, would make it less likely that the var gets tied into other configuration.

Now again, this is a best practice, if we don't follow this principle here I don't care that much cause this project is about move fast and get out the door and this definitely helps that.

Reply:
Ye it's updated now to specify if it's a local driver or a remote driver